CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+5.6.7 (September 2021)
+----------------------
+
+- Fix tablesfacility ([#58](https://github.com/ome/omero-gateway-java/pull/58))
+- Add method to get PlaneInfos ([#57](https://github.com/ome/omero-gateway-java/pull/57))
```

Comment:
Probably too late for this round as the release was cut butdo we  want to start switching to minor version increments to capture backwards compatible API additions?

Reply:
probably time now that it is a bit more mature